xmovie/presets.py

```diff
@@ -64,9 +64,9 @@ def _core_plot(ax, data, plotmethod=None, **kwargs):
     return p
 
 
-def _base_plot(ax, base_data, timestamp, plotmethod=None, **kwargs):
+def _base_plot(ax, base_data, timestamp, framedim, plotmethod=None, **kwargs):
     # need to convert time to input variable
```

Comment:
I think that this comment was my attempt to eventually fix this 🤦‍♂️. You could probably remove it!

Reply:
Hey at least it was documented! Actually this confirmed for me where the change needed to happen. Either way, done!